upload-service/templates/deployment.yaml

```diff
@@ -62,6 +62,69 @@ spec:
             value: {{ .Values.appConfig.config_center_base_url | quote }}
           - name: env
             value: {{ .Values.appConfig.env | quote }}
+          #if core or greenroom
+{{- if contains "core" .Values.appConfig.namespace }}
+          - name: ROOT_PATH
+            value: "/core-data"
+{{- else }}
+          - name: ROOT_PATH
+            value: "/data/core-storage"
+{{- end}}
+          - name: CORE_ZONE_LABEL
+            value: "Core"
+          - name: GREEN_ZONE_LABEL
+            value: "Greenroom"
+          #if tenant id is not set
+{{- if not .Values.appConfig.tenant_id }}
+          - name: DATAOPS_SERVICE
+            value: "http://dataops.utility:5065"
+          - name: PROJECT_SERVICE
+            value: "http://project.utility:5065"
+          - name: METADATA_SERVICE
+            value: "http://metadata.utility:5065"
+          - name: KAFKA_URL
+            value: "kafka.utility:9092"
+          - name: KAFKA_ACTIVITY_TOPIC
+            value: "metadata.items.activity"
+{{- else }}
+          - name: DATAOPS_SERVICE
+            value: "http://dataops.{{ .Values.appConfig.tenant_id }}-utility:5065"
+          - name: PROJECT_SERVICE
+            value: "http://project.{{ .Values.appConfig.tenant_id }}-utility:5065"
+          - name: METADATA_SERVICE
+            value: "http://metadata.{{ .Values.appConfig.tenant_id }}-utility:5065"
+          - name: KAFKA_URL
+            value: "shared-kafka.datastores-helmfile:9092"
+          - name: KAFKA_ACTIVITY_TOPIC
+            value: "metadata.items.{{ .Values.appConfig.tenant_id }}.activity"
```

Comment:
as we agreed, `tenant_id` should be used as a prefix. `{{ .Values.appConfig.tenant_id}}.metadata.items.activity`